python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -1650,6 +1650,16 @@ def do_evaluate(cls, schema: Schema, df: DataFrame) -> DataFrame:
         return DataFrame(columns)
 
 
+class MergeSorted(IR):
+    """Merge sorted operation."""
+
+    def __init__(self, schema: Schema, left: IR, right: IR, key: str):
+        # libcudf merge is not stable wrt order of inputs, since
+        # it uses a priority queue to manage the tables it produces.
+        # See: https://github.com/rapidsai/cudf/issues/16010
+        raise NotImplementedError("MergeSorted not yet implemented")
```

Comment:
Just to note (this comment is fine for now still), but I think polars no longer guarantees ordering for `merge_sorted`, so we can probably now implement this fine.

Reply:
xref #18026